ddl/index.go

```diff
@@ -255,11 +255,11 @@ func (d *ddl) onCreateIndex(t *meta.Meta, job *model.Job) error {
 		err = d.runReorgJob(func() error {
 			return d.addTableIndex(tbl, indexInfo, reorgInfo, job)
 		})
-		if terror.ErrorEqual(err, errWaitReorgTimeout) {
-			// if timeout, we should return, check for the owner and re-wait job done.
-			return nil
-		}
 		if err != nil {
```

Comment:
If `filtrateError(err, errWaitReorgTimeout)== nil` is true, we don't know if we need to return.